samples/Azure.AI.OpenAI/Generated/Docs/OpenAIClient.xml

```diff
@@ -0,0 +1,505 @@
+<?xml version="1.0" encoding="utf-8"?>
+<doc>
+  <members>
+    <member name="GetEmbeddingsAsync(String,RequestContent,RequestContext)">
+<example>
+This sample shows how to call GetEmbeddingsAsync with required parameters and request content, and how to parse the result.
+<code><![CDATA[
+var credential = new AzureKeyCredential("<key>");
+var endpoint = new Uri("<https://my-service.azure.com>");
+var client = new OpenAIClient(endpoint, credential);
+
+var data = new {
+    input = new {},
+};
+
+Response response = await client.GetEmbeddingsAsync("<deploymentId>", RequestContent.Create(data));
+
+dynamic result = response.Content.ToDynamicFromJson();
```

Comment:
We will need to pass options into the `ToDynamicFromJson()` call, or PascalCase getters will fail when the JSON is camelCased.  Could we pass [DynamicDataNameMapping.PascalCaseGettersCamelCaseSetters](https://apiview.dev/Assemblies/Review/b67e68cfee4841bda045f895c90ad383#Azure.Core.Dynamic.DynamicDataNameMapping)?

Reply:
Is it different than `DynamicJsonOptions.Default`?

Reply:
It has the same behavior.  I am considering removing `DynamicJsonOptions` from the final API when we GA the type.